tezos/timing/src/lib.rs

```diff
@@ -259,15 +262,22 @@ pub static TIMING_CHANNEL: Lazy<Sender<TimingMessage>> = Lazy::new(|| {
 });
 
 fn start_timing(recv: Receiver<TimingMessage>) {
-    #[cfg(not(test))]
-    let db_path = Some(DB_PATH.into());
+    let mut msgs = Vec::new();
```

Comment:
@sebastiencs I am confused about the logic here. I we are not timing, we want to discard everything, not collecting it right? if timings are never initiated, we will have a memory leak here.

Reply:
I would consider such situation a bug, and we can probably error out in that case (or log a warning). But lets revise it later once this has been merged and I have updated the OCaml side. For now lets just discard the messages.